### DISCUSSION TITLE
I would like to migrate my license from Chrome to Firefox.

### DISCUSSION THREAD

**Discussion by squareform** [aclap-dev/video-downloadhelper#1551] [License & Purchase requests]
### What's your issue?

Hi,
I would like to migrate my license from Chrome to Firefox.
Is it possible and what do I have to do?

Best
Wojtek

### The 10 first digits (only 10) of your license number or email address

51f4efba5b

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Sorry we can't swap licenses.
Chrome and Firefox premium are separate products with different prices and features.
What I can do is refund your Chrome premium so that you can purchase Firefox premium if you want to.
Do you want me to do this ?

**Reply by squareform** [reply]
Hi,
thanks for your reply. 
OK, so I would like to take the refund to purchase Firefox Premium!

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Reply by squareform** [reply]
Thanks!!!
